frontend/src/components/activity/content/Storyboard.vue

```diff
@@ -165,8 +108,17 @@ export default {
     isLastSection() {
       return Object.keys(this.sections).length === 1
     },
+    variant() {
+      return this.clientWidth <= 910 ? 'dense' : 'default'
```

Comment:
pity that it did not work with css only.
Are media queries (or an abstraction of media query) not the way to go for this problem anymore?

Reply:
If someday we have support for container queries, then this could be doable